.travis.yml

```diff
@@ -14,15 +14,33 @@ env:
   - DIR=19
   - DIR=19 VARIANT=slim
 
+install:
+  - git clone https://github.com/docker-library/official-images.git ~/official-images
+
 before_script:
   - cd "$DIR"
   - eval $(awk '/OTP_VERSION=/ { sub(/@/, "-", $2); print $2; exit }' ${VARIANT:-.}/Dockerfile)
   - image="erlang:${OTP_VERSION}${VARIANT:+-$VARIANT}"
 
 script:
-  - docker build --pull -t "$image" "${VARIANT:-.}"
-  - curl -fsSL https://github.com/docker-library/official-images/archive/master.tar.gz | tar -zxvv official-images-master/test/
-  - ./official-images-master/test/run.sh "$image"
+    - |
+      (
+      set -Eeuo pipefail
+      set -x
```

Comment:
why not run the two `set` at same line? except some options are super popular `set -x -e` I think it's better to use the all verbose options if you use some unfamiliar ones, like `set -o errtrace -o errexit -o nounset -o pipefail -o xtrace` ??

```console
$ help set
set: set [-abefhkmnptuvxBCHP] [-o option-name] [--] [arg ...]
    Set or unset values of shell options and positional parameters.
    
    Change the value of shell attributes and positional parameters, or
    display the names and values of shell variables.
    
    Options:
      -a  Mark variables which are modified or created for export.
      -b  Notify of job termination immediately.
      -e  Exit immediately if a command exits with a non-zero status.
      -f  Disable file name generation (globbing).
      -h  Remember the location of commands as they are looked up.
      -k  All assignment arguments are placed in the environment for a
          command, not just those that precede the command name.
      -m  Job control is enabled.
      -n  Read commands but do not execute them.
      -o option-name
          Set the variable corresponding to option-name:
              allexport    same as -a
              braceexpand  same as -B
              emacs        use an emacs-style line editing interface
              errexit      same as -e
              errtrace     same as -E
```